apps/common/views/partials/add-another-address-summary.html

```diff
@@ -1,5 +1,5 @@
 {{#hasAddresses}}
-  <div id="storage-address-summary">
+  <div id="address-summary">
     <h2>Addresses added</h2>
```

Comment:
@joefitter It's in my work that is the second part of this. So it will be in my pr this